CONTRIBUTING.md

```diff
@@ -27,13 +47,13 @@ brew install --HEAD -s dotnet-sdk.rb
 rm dotnet-sdk.rb
 ```
 
-## Hacking on Pulumi in Gitpod
+### Working on Pulumi in Gitpod
 
 If you have a web browser, you can get a fully pre-configured Pulumi development environment in one click:
 
 [![Open in Gitpod](https://gitpod.io/button/open-in-gitpod.svg)](https://gitpod.io/#https://github.com/pulumi/pulumi)
 
-## Make build system
+### Make build system
 
 We use `make` as our build system, so you'll want to install that as well, if you don't have it already. We have extremely limited support for doing development on Windows (the bare minimum for us to get Windows validation of `pulumi`) so if you're on windows, we recommend that you use the [Windows Subsystem for Linux](https://docs.microsoft.com/en-us/windows/wsl/install-win10). We'd like to [make this better](https://github.com/pulumi/pulumi/issues/208) so feel free to pitch in if you can.
```

Comment:
This definitely still feels accurate. I've put Windows testing & build system improvements on the backburner for this iteration because it's a time sink, but I expect @t0yv0 and I will return to it.

Reply:
The WSL install URL now redirects to just https://docs.microsoft.com/en-us/windows/wsl/install which gives instructions for Windows 10 & Windows 11.